src/binary-reader.cc

```diff
@@ -66,12 +66,20 @@ namespace {
 
 class BinaryReader {
  public:
+  struct ReadModuleOptions {
+    ReadModuleOptions() = default;
```

Comment:
IMO we could keep this a little simpler by relying on brace-initializer syntax instead, see https://godbolt.org/z/je9sT5Eo5

Reply:
This has been addressed. I have ran clang-format to double check that the formatting is fine. I have not found any other linters `wabt` seems to use, so I'll have to wait for CI approval to confirm if everything is fine.